src/config/backends/config-fs.ts

```diff
@@ -116,7 +112,10 @@ export class ConfigFs extends ConfigBackend {
 	}
 
 	private async writeConfigJSON(config: ConfigfsConfig): Promise<void> {
-		await remountAndWriteAtomic(this.ConfigFilePath, JSON.stringify(config));
+		await hostUtils.writeToBootAbsolute(
+			this.ConfigFilePath,
+			JSON.stringify(config),
+		);
```

Comment:
I do agree that the method is unnecessary, but I think I'll keep it, since it the next step in this improvement is to move to https://github.com/balena-os/fatrw. When that happens, all we will need is to modify this single method to have all writes on boot adopt this new secure writing mechanism.

I will rename it to `writeToBoot` as the existing `writeToBoot` method is not used.